meson.build

```diff
@@ -135,8 +135,15 @@ else
   valgrind = dependency('valgrindXXX', required: false)
 endif
 soup = dependency('libsoup-2.4', version : '>= 2.51.92')
-gpgme = cc.find_library('gpgme')
-gpgerror = cc.find_library('gpg-error')
+if get_option('enable-pkcs7')
+  gnutls = dependency('gnutls')
+  conf.set('ENABLE_PKCS7', '1')
+endif
+if get_option('enable-gpg')
+  gpgme = cc.find_library('gpgme')
+  gpgerror = cc.find_library('gpg-error')
+  conf.set('ENABLE_GPG', '1')
+endif
```

Comment:
Is it a problem if both are turned off? If we turn both off and then try to use a GPG-signed remote it's just going to fail -- if it's a locked-down IOT device I think it'd be fine to have no signing required.